app/controllers/admin/comments/add_to_blocklist_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Admin
+  module Comments
+    class AddToBlocklistController < Admin::BaseController
```

Comment:
Couple thoughts, you can hit me up in slack to hash it out:

1. I think we'll need to add/remove/index the blocklist (so the controller should be probably named `BlocklistController`)

2. It's IPs that we are managing (not comments) and we'll also be blocking them based on tracks, users, etc — so I'm thinking we probably want it to be `Admin::BlocklistController`